Sources/MarathonCore/PackageManager.swift

```diff
@@ -364,6 +364,7 @@ public final class PackageManager {
                           "import PackageDescription\n\n" +
                           "let package = Package(\n" +
                           "    name: \"\(masterPackageName)\",\n" +
+                          "    products: [.library(name: \"Dependencies\", type: .dynamic, targets: [\"\(masterPackageName)\"])],\n" +
```

Comment:
What do you think about calling this `MarathonDependencies` to be a bit more clear where it comes from?

Reply:
Good idea!